terms/experimentalData/stemCellBaseMedium.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "sage.annotations-experimentalData.stemCellBaseMedium-0.0.1",
+  "description": "Culture medium that stem cells are grown in during differentiation.",
+  "type": "string",
+  "minLength": 1,
+  "maxLength": 50
+}
+
```

Comment:
```suggestion
```
It looks like there might be two newlines at the end of this file. This suggestion should remove one of the two (only one is needed for the file to open and be read correctly).